Resources/doc/reference/security.rst

```diff
@@ -7,7 +7,7 @@ a download strategy interface, which can be set per context and authorize the me
 
 Built-in security strategy:
 
-* ``sonata.media.security.superadmin_strategy`` : DEFAULT - the user need to have one of the following roles : ``ROLE_SUPER_ADMIN`` or ``ROLE_ADMIN``
+* ``sonata.media.security.superadmin_strategy`` : DEFAULT - the user need to have one of the following roles : ``ROLE_SUPER_ADMIN`` or ``ROLE_ADMIN`` (although these roles can be configured in SonataAdminBundle)
```

Comment:
"need" => "needs"

Reply:
Please comply with the sf docs standards:

> [Each line should break approximately after the first word that crosses the 72nd character (so most lines end up being 72-78 characters);](http://symfony.com/doc/current/contributing/documentation/standards.html)

Reply:
Not my fault but yes I can change it :)

Reply:
Indeed! Thanks for applying the boy-scout rule!